sdk/eventhub/event-hubs/test/eventProcessor.spec.ts

```diff
@@ -735,6 +749,133 @@ describe("Event Processor", function(): void {
       partitionOwnershipMap.get(processorByName[`processor-0`].id)!.length.should.oneOf([n, n + 1]);
       partitionOwnershipMap.get(processorByName[`processor-1`].id)!.length.should.oneOf([n, n + 1]);
     });
+
+    it("should ensure that all the processors maintain a steady-state when all partitions are being processed", async function(): Promise<
+      void
+    > {
+      const partitionIds = await client.getPartitionIds({});
+      const checkpointStore = new InMemoryCheckpointStore();
+      const claimedPartitionsMap = {} as { [eventProcessorId: string]: Set<string> };
+
+      let allPartitionsClaimed = false;
+      let thrashAfterSettling = false;
+      const handlers: SubscriptionEventHandlers = {
```

Comment:
I'm not sure if we'd get enough value compared to the work of maintaining a list of errors to filter out as we update checkpointstore.

We'll see errors related to losing ownership, and errors when the processor fails to claim ownership (which might get updated as we update the checkpointstore), but in neither of these cases (assuming inMemoryCheckpointStore) are these errors we care about since they're expected while the processors are becoming balanced.

We'll miss out on catching service connection issues but that should show up in thrashing (or other tests) so I'm not as worried about that.